### DISCUSSION TITLE
nice-article/other/jugzfjtdsjysxmdnicemd

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#207] [Announcements]
# nice-article/other/jugzfjtdsjysxmdnicemd

支付中心存在的目的支付中心系统对内为各个业务线提供统一的支付、退款等服务，对外对接三方支付或银行服务实现资金的流转。如下图：大部分公司基本都是这样的架构，主要有以下几方面的优点：形成统一支付服务，降低

https://tobebetterjavaer.com/nice-article/other/jugzfjtdsjysxmdnicemd.html

**Comment by wsgzao** [comment]
博主的手绘风格流程图是怎么制作的呀
